Packs/Troubleshoot/Scripts/CertificatesTroubleshoot/CertificatesTroubleshoot.py

```diff
@@ -153,39 +154,53 @@ def docker_container_details() -> dict:
             1. Global veriables which used by requests module:
                 a. SSL_CERT_FILE
                 b. REQUESTS_CA_BUNDLE
-            2. Custom python ssl file located in docker container - /etc/custom-python-ssl/certs.pem
+            2. Custom python ssl file located in docker container - SSL_CERT_FILE
 
     Returns:
         dict: Corresponding enrty context.
     """
-    container_ca_file = Path('/etc/custom-python-ssl/certs.pem')
-    certificates = "" if not container_ca_file.is_file() else container_ca_file.read_text()
+    ssl_cert_file_env = os.environ.get('SSL_CERT_FILE')
+    custom_certs = []
+    if ssl_cert_file_env:
+        container_ca_file = Path(ssl_cert_file_env)
+        certificates = "" if not container_ca_file.is_file() else container_ca_file.read_text()
+        custom_certs = parse_all_certificates(certificates)
+        demisto.debug(f'custom certs len: {len(custom_certs)}')
     return {
         "ShellVariables": {
             "SSL_CERT_FILE": os.environ.get('SSL_CERT_FILE'),
             "CERT_FILE": os.environ.get('REQUESTS_CA_BUNDLE'),
         },
-        "CustomCertificateAuthorities": parse_all_certificates(certificates),
+        "CustomCertificateAuthorities": custom_certs[:5],
```

Comment:
shouldnt this be configurable ?

Reply:
I think it's ok. It's for safety not to overloaded the response. I don't want to over engineer things.